src/Symfony/Component/Yaml/Parser.php

```diff
@@ -359,6 +308,47 @@ public function parse($value, $flags = 0)
             mb_internal_encoding($mbEncoding);
         }
 
+        if (array_key_exists('<<', $data)) {
+            $merged = $data['<<'];
```

Comment:
I'm not sure this is worth it, the check would have to be added in two places while it is only in one now.